libs/fabric/src/lib/components/dropdown/directives/dropdown-options.directive.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { ContentChildren, Directive, QueryList } from '@angular/core';
+
+import { DropdownOptionDirective } from './dropdown-option.directive';
+import { IDropdownOption } from 'office-ui-fabric-react';
+
+/**
+ * Wrapper directive for creating multiple DropdownOptions
+ */
+@Directive({ selector: 'fab-dropdown > options' })
+export class DropdownOptionsDirective {
+  @ContentChildren(DropdownOptionDirective) readonly directiveItems: QueryList<DropdownOptionDirective>;
```

Comment:
maybe make this optional (and verify it exists in the `items` getter below, to make sure we catch the empty case:
```html
<fab-dropdown>
  <options></options>
</fab-dropdown>
```

Reply:
The map handles this correctly. You just wind up with a slector that has no dropdown options.